ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,179 @@
+145.7 "Fizzling Duster" sync / 1[56]:[^:]*:Silkie:7765:/
+146.0 "Soapsud Static" #sync / 1[56]:[^:]*:Silkie:77ED:/
+160.1 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+166.2 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+172.3 "Eastern Ewers" sync / 1[56]:[^:]*:Silkie:776D:/
+178.6 "Rinse" sync / 1[56]:[^:]*:Eastern Ewer:776F:/ duration 7.3
+188.1 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+188.1 "Bracing Duster and Fizzling Duster" #sync / 1[56]:[^:]*:Silken Puff:(7769|776A):/
+195.2 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+202.3 "Carpet Beater" sync / 1[56]:[^:]*:Silkie:774F:/
+204.4 "--sync--" sync / 1[56]:[^:]*:Eastern Ewer:79FB:/
+209.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+209.8 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:7760:/
+213.4 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+225.9 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+233.0 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+233.0 "Chilling Duster and Bracing Duster" #sync / 1[56]:[^:]*:Silken Puff:(7768|7769):/
+244.2 "Dust Bluster" sync / 1[56]:[^:]*:Silkie:776C:/
+249.6 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+255.7 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+262.3 "Squeaky Clean" sync / 1[56]:[^:]*:Silkie:(7751|7752):/
+263.8 "Squeaky Clean 1" sync / 1[56]:[^:]*:Silkie:7753:/
+265.5 "Squeaky Clean 2" sync / 1[56]:[^:]*:Silkie:7754:/
+266.9 "Puff and Tumble" #sync / 1[56]:[^:]*:Silken Puff:7771:/
+266.9 "Squeaky Clean 3" sync / 1[56]:[^:]*:Silkie:(7755|7756):/
+276.0 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+276.0 "Chilling Duster and Bracing Duster" #sync / 1[56]:[^:]*:Silken Puff:(7768|7769):/
+283.1 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+292.3 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+294.4 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+299.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+299.9 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:7760:/
+303.5 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+309.0 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+311.1 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+316.1 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+316.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:7760:/
+320.0 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+332.9 "Total Wash (enrage)" sync / 1[56]:[^:]*:Silkie:79E0:/
+338.1 "Total Wash (enrage)" sync / 1[56]:[^:]*:Silkie:79F9:/
+
+#~~~~~~~~~~~~~~~~~~~~~~#
+# Gladiator of Sil'dih #
+#~~~~~~~~~~~~~~~~~~~~~~#
+
+# -ii 766F 7670 7676 7677 77BF 7660
+
+1000.0 "--sync--" sync / 00:0839::The Trial of Might will be sealed off/ window 1000,1
+1006.8 "--sync--" sync / 14:[^:]*:Gladiator of Sil'dih:7671:/ window 1007.3,5
+1011.5 "Flash of Steel" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7671:/
+1023.6 "Specter of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7673:/
+1038.3 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765B:/
+1040.4 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765C:/
```

Comment:
Same comment here, re: remove the damage line and keep the self-cast line.

Reply:
Hmm... I was thinking one is the first Cleave and the other is the second Cleave, not damage lines.

Reply:
Oh, if it's first and second, then it should be "Rush of Might (near)" and "Rush of Might (far)" kinda thing? or even "1" and "2" vs just two identically named abilities back to back.

Reply:
Makes sense. It could be 1 or 2.
There is also the possibility of stating the safe spot in it, but I think that doesn't work if it happens to be 2/3 or 3/2 pattern.